softdev.bib

```diff
@@ -711,11 +711,12 @@ @phdthesis{visser97sdf
     school = {University of Amsterdam}
 }
 
-@book{visser97scannerless,
-    title={Scannerless generalized-LR parsing},
-    author={Visser, Eelco and others},
-    year={1997},
-    publisher={Universiteit van Amsterdam. Programming Research Group}
+@techreport{visser97scannerless,
+    title = {Scannerless generalized-LR parsing},
+    author = {Visser, Eelco and others},
+    number = {P9707}
+    year = {1997},
+    publisher = {Universiteit van Amsterdam. Programming Research Group}
```

Comment:
Is the publisher actually the PRG? Or is it a university wide technical report? [Typically it's either university wide or department wide. I've never seen it be research group wide, but maybe that's just me leading a sheltered life!]

Reply:
Really hard to tell. The only info I found is this: https://www.narcis.nl/publication/RecordID/oai:dare.uva.nl:publications%2F15977da5-73a9-4d9a-b7d9-f215630dc31d

Reply:
I can't find anything obvious either. I can find a paper with Eelco as an author where he uses the same publisher as you do -- perhaps that's as good as we'll manage! Let's stick with this then.